components/layout/ActivityList.js

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+import ActivityCard from "@/components/layout/ActivityCard";
+import ActivityForm from "@/components/layout/ActivityCreateForm";
+
+export default function ActivityList({ activities, handleActivities }) {
+  return (
+    <>
+      <ActivityForm
+        activities={activities}
+        onHandleActivities={handleActivities}
+      />
```

Comment:
It's better to separate the concerns. The list component should only return a list, not a form and a list. If you want anywhere in your app a form and a list you should render first the form and then the list. This way each component is better reusable.